server/package.json

```diff
@@ -4,10 +4,10 @@
   "description": "Cognitive CRM Server",
   "main": "server.ts",
   "scripts": {
-    "start:prod": "node ./node_modules/typescript/lib/tsc && export NODE_ENV=production && node dist/server.js",
+    "start:prod": "node ./node_modules/typescript/lib/tsc && NODE_ENV=production && node dist/server.js",
     "start": "npm run build:live",
     "build": "node ./node_modules/typescript/lib/tsc",
-    "build:live": "nodemon --exec ./node_modules/.bin/ts-node -- server.ts",
+    "build:live": "NODE_ENV=development && nodemon --exec ./node_modules/.bin/ts-node -- server.ts",
```

Comment:
I don't think you want the "&&" here and above.  It probably works find, but I think the better way to set and env for command is just "VAR=foo nodemon...".  Less side-effectey.